### DISCUSSION TITLE
DBT Run Multi-tag Selector is not working

### DISCUSSION THREAD

**Discussion by mytreyireddy** [dbt-labs/dbt-core#5328] [Community]
Hi Everyone,

There is a use-case to run dbt command with a multi tag such as

`dbt run --select tag:products, tag:orders` or `dbt run -m tag:products, tag:orders`

But the above command is executing only the second tag related tables and not executing the first tag related tables.

Could anyone please guide in executing the mult--tag sql models in DBT ?

**Comment by jwills** [comment]
Mmm, I don't think you want to have commas in between the tags that you want to select, assuming that you want the union of the models that have `tag:products` and `tag:orders`. See here: https://docs.getdbt.com/reference/node-selection/set-operators/

**Reply by mytreyireddy** [reply]
I executed the commands without comma but still the `tag:orders` is only executed.  The `tag:products` is not executed.
`dbt run -m tag:products tag:orders`

**Comment by jwills** [comment]
Hrm, then I'm not sure what's wrong; if you think that it's a bug or a regression, you want to file an [issue](https://github.com/dbt-labs/dbt-core/issues) or post a help question in the [dbt Slack](https://www.getdbt.com/community/join-the-community/).

**Reply by mytreyireddy** [reply]
Sure will file an issue! Thank you !

**Reply by ManishJaiswal007** [reply]
Hi @mytreyireddy, have you find solution for this ?

**Reply by mytreyireddy** [reply]
No @ManishJaiswal007. Multi-tag selector didnt work.

**Reply by clay-walker** [reply]
> Sure will file an issue! Thank you !

@mytreyireddy Did you file an issue? I'd like to contribute as this is also something I need.

**Reply by mytreyireddy** [reply]
I didnt file an issue on multi-tag. @clay-walker

**Reply by karlkaebnick** [reply]
Checking in on this - did support for this get added? (or an issue opened?)

It might be nice to be able to control whether the multiple tags select models that have *all* the tags, or models that have *any* of the tags, with a syntax like:

- to select models with all of the tags: `select=tag1+tag2`
- to select models with any of the tags: `select=tag1|tag2`

**Reply by mytreyireddy** [reply]
@karlkaebnick I created the issue: https://github.com/dbt-labs/dbt-core/issues/7129
